packages/pages/src/dev/server/server.ts

```diff
@@ -31,6 +32,30 @@ export const createServer = async (
     },
   });
 
+  // Read features.json and set the default locale to the first locale listed
+  // Default to en if features.json cannot be read or there is no locales entry
+  let defaultLocale = "en";
+  try {
+    const featuresJson = JSON.parse(
+      fs.readFileSync(
+        path.join(
+          projectStructure.sitesConfigRoot.getAbsolutePath(),
+          projectStructure.featuresConfig
+        ),
+        "utf-8"
+      )
+    );
+    if (featuresJson.locales && featuresJson.locales.length > 0) {
+      defaultLocale = featuresJson.locales[0];
+    }
+  } catch (e) {
+    if (e === "Error: ENOENT: no such file or directory") {
+      console.warn("Could not find features.json");
```

Comment:
This is going to be the case most of the time so let's not log anything in this case.